user-manager/src/test/java/com/qpay/usermanager/mapper/CustomerMapperTest.java

```diff
@@ -0,0 +1,36 @@
+package com.qpay.usermanager.mapper;
+
+import com.qpay.usermanager.model.dto.customer.CustomerModification;
+import com.qpay.usermanager.model.entity.customer.CustomerEntity;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CustomerMapperTest {
+
+    private final CustomerMapper CUSTOMER_MAPPER = new CustomerMapper();
```

Comment:
it is not a constant static variable to describe smth like string, number and so on. It's your actual object for real work. `customerMapper` use standard naming convention